src/NAR/Constructs/SquareRootConstruct.java

```diff
@@ -0,0 +1,117 @@
+package NAR.Constructs;
+
+import NAR.Editor;
+import NAR.HighLevelFunc;
+import NAR.Parser;
+
+public class SquareRootConstruct extends HighLevelFunc {
+    private double op1, op2, result;
+
+    public SquareRootConstruct() {
+        super("square");//putting square root gives unknown construct
```

Comment:
please enter square root for the name since I pushed a fix so the construct would be recognized